tutorials-v5/time-evolution/001_qobjevo.md

```diff
@@ -221,34 +247,30 @@ td_args_str(np.pi)
 
 ### Dynamic arguments
 
-When `QobjEvo` is used in the solvers, certain dynamic arguments will be populated at each iteration, if and only if their names are present in the `args` dictionary used at `QobjEvo` intialisation.  The initial values of all of these dynamic arguments will be a representation of `0` in the corresponding type, for example `"state"` will be `qzero()` of the correct dimensions.
+When `QobjEvo` is used in the solvers, the solver states, derivated values or other internal values can be made available as arguments.
+
+These values are set from method to the solver classes:
+
+Most solver support:
 
-There several of these "magic" variables, mostly revolving around the state currently being evolved:
-  - `"state"` or `"state_qobj"`: a `Qobj` of the current state.
-  - `"state_mat"`: a dense 2D `np.ndarray` of the state as a matrix, similar to `state.full()`
-  - `"state_vec"`: a dense 1D `np.ndarray` of the state as a vector.  This only generally makes sense for kets.
-  - `"expect_op_<n>"`: `complex`, where `<n>` is an index into `e_ops`, the current expectation value of `e_ops[n]` (the `<>` should not appear, e.g. `"expect_op_0"`).
-  - `"collapse"`: a `list` of `(t: float, n: int)` indicating the time `t` a collapse occurred, and which of `c_ops` caused it.  Only present when using `mcsolve`.
+  - `StateFeedback`: state as a `Qobj` or qutip `Data`.
+  - `ExpectFeedback`: Expectation value computed from the state.
 
+Additionnally `mcsolve` has `CollapseFeedback` to get the collaspe list and stochastic solvers have `WienerFeedback` that return the Wiener function along the trajectory.
+
+They all take a `default` input that set the value when used outside a solver. This value must be valid as it will be used during setup.
```

Comment:
```suggestion
They all take a `default` input that specifies the value to be used when the `QobjEvo` is accessed outside of a solver. This value must be a valid input for the `QobjEvo` -- it will be used during solver setup.
```